<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DrawerCloser</class>
 <widget class="QDialog" name="DrawerCloser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1370</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Drawer Closer</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Receipts made since last close : </string>
   </property>
  </widget>
  <widget class="QTableWidget" name="receiptTable">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>1051</width>
     <height>501</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>581</height>
    </rect>
   </property>
   <property name="title">
    <string>Drawer Information : </string>
   </property>
   <widget class="QLabel" name="lastCloseDate">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>_____________________</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Last Close Date Time : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cash from sales in drawer :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="cashInDrawer">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>___________</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="comments">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>251</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>550</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Charachters left:</string>
    </property>
   </widget>
   <widget class="QLabel" name="remainingCommentCharachterLabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>550</y>
      <width>46</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>_______</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Amount of cash actually in drawer :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="amountFound">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>200</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total in checks :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="checksInDrawer">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>___________</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Chargebacks :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="chargebacksInDrawer">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>___________</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Subtotal : </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="subtotalInDrawer">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>___________</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="viewDetails">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>540</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>View Receipt Details</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeDrawer">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>540</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Close Drawer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeCloseButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>540</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Selected From Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
